Installation/include/CGAL/config.h

```diff
@@ -463,4 +463,11 @@ typedef const void * Nullptr_t;   // Anticipate C++0x's std::nullptr_t
 
 } //namespace CGAL
 
+//Support for c++11 noexcept
+#ifdef CGAL_CXX11
```

Comment:
This line should be replaced by:
`#if (BOOST_VERSION > 105000 && !defined(BOOST_NO_CXX11_NOEXCEPT) ) ||
      (BOOST_VERSION > 104600 && BOOST_VERSION < 105100 && !defined(BOOST_NO_NOEXCEPT) )`